guides/NPM_PACKAGE.md

```diff
@@ -0,0 +1,89 @@
+# npm package
+
+This repository can be installed as an npm package. The goal is to make all of the Node.js project's English and localized content more easily consumable and re-usable in different projects.
```

Comment:
```suggestion
This repository can be installed as an npm package. The goal is to make the English and localized Node.js docs more easily consumable and re-usable in different projects.
```

because, this module provides only docs, not a content for websites